content/guides/guides/debugging.md

```diff
@@ -47,18 +59,36 @@ will return immediately, having enqueued their work to be done later, and
 Let's use [`.then()`](/api/commands/then) to tap into the Cypress command during
 execution and add a `debugger` at the appropriate time:
 
+:::visit-mount-test-example
```

Comment:
Please ensure the content after this example mentions `cy.mount()` where relevant.

Reply:
This is covered in https://github.com/cypress-io/cypress-documentation/pull/4207/files